src/content/guides/asset-modules.md

```diff
@@ -26,6 +26,8 @@ Asset Modules type replaces all of these loaders by adding 4 new module types:
 - `asset/source` exports the source code of the asset. Previously achievable by using `raw-loader`.
 - `asset` automatically chooses between exporting a data URI and emitting a separate file. Previously achievable by using `url-loader` with asset size limit.
 
+W>When using the old assets loaders in webpack 5 you might want to exclude them from the assets module processing to prevent asset duplication. This can be done by overriding the asset configuration for that asset type by setting it's module type to "javascript/auto". To exclude assets that came from new URL calls from the asset loaders add dependency: { not: "url" } to the loader configuration.
```

Comment:
Let's clarify what is `old assets loaders` - i.e. `file-loader`/`url-loader`/`raw-loader` and add simple examples for `javascript/auto` and `dependency: { not: "url" }`

Reply:
I added examples and removed the W> because I can't see how it will rendered.